src/http-client.ts

```diff
@@ -51,7 +51,20 @@ export class HttpClient {
     try {
       return await response.send(opts.data); // trigger request
     } catch (err) {
-      throw this.formatRequestError(err);
+      const res = err.response;
+
+      // Potentially contains PII information - Do not send to Sumo, just log to console.
+      throw {
+        status: err.status,
+        correlationId: res.headers['inin-correlation-id'],
+        responseBody: res.text,
+        requestBody: res.req._data,
```

Comment:
I'm not sure we can/should log the requestBody as it could contain PII.

Reply:
I'll remove it